FeatureTest.java

```diff
@@ -6,7 +6,8 @@ public static void main(String[] args) {
     public int square(int variable){
         String str = null;
         System.out.println(str.length());
-        str = "fjff";
```

Comment:
Use a logging framework like SLF4J and log the NullPointerException at ERROR level with relevant context.